lib/options-parser.js

```diff
@@ -44,6 +46,7 @@ function parseOptions(argsOriginal) {
     parsed.url = args[0];
     parsed.css = args[1];
 
+    // this is problematic...
```

Comment:
The below guard fails the tests for [critical](https://github.com/addyosmani/critical/blob/master/test/test.js#L50), as they're passing relive paths to `Penthouse`, f.e. `test/fixture/index.html`.

I just looked quickly and couldn't see how they get away with doing that, as phantomjs will always fail to connect to URL's when I leave out the `protocol`...

Since it must be possible to do so in some way, other people might be doing it as well, and I therefor think we have to drop the check below completely.. If it errors, it will be caught inside the core script anyway.